zvm-stdlib/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "zvm-stdlib"
+version = "0.1.0"
+authors = ["iximeow <me@iximeow.net>"]
+edition = "2018"
+description = "zvm goes zoom zoom"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+jni = "0.14.0"
```

Comment:
let's defer the jni dependency until a PR that uses it (by my count, that also includes deferring the Cargo.lock changes)